src/group.ts

```diff
@@ -12,7 +12,7 @@ function txGroupPreimage(txnHashes: Uint8Array[]): Uint8Array {
       `${txnHashes.length} transactions grouped together but max group size is ${ALGORAND_MAX_TX_GROUP_SIZE}`
     );
   }
-  const bytes = encoding.encode({
+  const bytes = encodeObj({
```

Comment:
isn't it deprecated now so that a replacement should be used?

Reply:
Good point, updated in 32c59407